src/cmd-init

```diff
@@ -197,6 +193,9 @@ if [[ -n "${VARIANT}" ]] && [[ "${VARIANT}" != "default" ]]; then
   "coreos-assembler.config-variant": "${VARIANT}"
 }
 EOF
+elif [[ ! -f "${manifest}" ]] || [[ ! -f "${image}" ]]; then
+    echo 1>&2 "Could not find default manifests (${manifest} & ${image})"
+    fatal "If you are using a custom configuration, be sure it has a manifest.yaml & image.yaml."
```

Comment:
I think we are changing the requirements here so we need to update the message:

```suggestion
    fatal "If you aren't using a custom configuration, be sure there are manifest.yaml & image.yaml files."
```

Reply:
I applied the change, but I think the "custom configuration" mentioned here is referring to a non-`fedora-coreos-config` configuration.  In that context, it hasn't changed behavior.  The `--variant` is a variant within a config that uses different non-default manifest and image YAML files.